source/manual/github-unavailable.html.md

```diff
@@ -23,9 +23,92 @@ We [mirror all GitHub repositories](repository-mirroring.html) tagged with `govu
 
 Use the normal deployment Jenkins job but check the box to deploy from AWS CodeCommit.
 
-### Making changes to code in AWS CodeCommit before deployment
+#### Making changes to code in AWS CodeCommit before deployment
 
-GOV.UK AWS admin users can give access to developers who need to make changes to the code before deployment. This may be necessary if we need to work in private, for example to fix a security vulnerability without disclosing it to the public. To do this, push to a new branch on AWS CodeCommit and then deploy that code.
+GOV.UK AWS admin users can give access to developers who need to make changes to the code before deployment.
+
+1. In the root of the local repo, run the following commands to install the AWS
+   credential helper and add CodeCommit as a remote:
+
+   ```
+   git config credential.helper '!aws codecommit credential-helper $@'
+   git config credential.UseHttpPath true
+   git remote add aws https://git-codecommit.eu-west-2.amazonaws.com/v1/repos/<app>
+   ```
+
+1. [Get some AWS credentials](/manual/deploying-terraform.html#2-get-your-credentials)
+   for the `govuk-tools` AWS account
+
+1. Export the access key ID, secret access key and session token from the last step,
+   for example:
+
+   ```
+   export AWS_ACCESS_KEY_ID=...
+   export AWS_SECRET_ACCESS_KEY=...
+   export AWS_SESSION_TOKEN=...
+   ```
+
+1. Fetch the AWS upstream by running `git fetch aws`
+
+1. Checkout a new branch on the upstream by running `git checkout -b aws/my-super-secret-fix`
+
+1. Make and commit your changes to this branch, and make sure all tests run successfully
+   locally (since CodeCommit does not run tests)
+
+1. Push your changes to CodeCommit by running `git push`
+
+1. Tag your changes by running `git tag release_XYZ`, where XYZ is one more that the latest
+   release tag for the application you're working on, as reported by the Release app
+
+1. Push your new tag to CodeCommit by running `git push aws release_XYZ`
+
+#### Deploying the code change
+
+1. Review the pull request on AWS CodeCommit.
```

Comment:
This might be an stupid question, but how does one do this?

Reply:
I have no idea.  I'll see if I can figure it out in the AWS Console.